docker-compose.yml

```diff
@@ -35,20 +35,34 @@ services:
       zookeeper:
         condition: service_healthy
 
-  trademodule-postgres:
-  ## TODO: Tag this image locally
-    image: "postgres:15-alpine"
+  trademodule-postgis:
+    ## TODO: Tag this image locally
+    image: "postgis/postgis:15-3.3-alpine"
     restart: on-failure
     env_file:
       - env/${ENVIRONMENT}/.env_trademodule
     volumes:
-      - "trademodule_postgres:/var/lib/postgresql/data"
+      - "trademodule_postgis:/var/lib/postgresql/data"
     healthcheck:
       test: ["CMD-SHELL", "pg_isready -d trademodule -U edpn_trademodule"]
       interval: 10s
       timeout: 5s
       retries: 5
 
+  explorationmodule_postgis:
+    ## TODO: Tag this image locally
```

Comment:
I like that you copied my TODO tags :) I'll get to those tags when I can, probably as we're next deploying to Morris.